packages/terra-docs/src/Button.module.scss

```diff
@@ -0,0 +1,39 @@
+@import './clinical-lowlight-theme/Button.module';
```

Comment:
I based the example button's styling off of the neutral button. @neilpfeiffer Should this button have more distinct styling to separate it from the example it's being used in?

Reply:
Is it possible to slim down these styles & what is themed so there is less to maintain?